frontend/src/app/candidates/candidate-list/candidate-form/candidate-form.component.ts

```diff
@@ -0,0 +1,140 @@
+import { Component, OnInit, Output, EventEmitter, OnDestroy } from '@angular/core';
+import { Candidate } from '../../../api/models/candidate';
+import { FormGroup, FormControl, FormBuilder, Validators, FormArray, ValidatorFn } from '@angular/forms';
+import { CandidateControllerService } from '../../../api/services/candidate-controller.service';
+import { CandidateService } from '../../service/candidate.service';
+import { Discipline } from '../../../api/models';
+import { Subject } from 'rxjs/Subject';
+import { DisciplineControllerService } from '../../../api/services';
+
+
+@Component({
+  selector: 'app-candidate-form',
+  templateUrl: './candidate-form.component.html',
+  styleUrls: ['./candidate-form.component.css']
+})
+export class CandidateFormComponent implements OnInit, OnDestroy {
+
+  @Output() displayForm = new EventEmitter();
+  public candidate: Candidate = new Candidate();
```

Comment:
'public' is not required.

Reply:
Yes, he is not required. I wanted use him in template, but now I using reactive forms. I will correct it.

Reply:
he -> it;